docs/src/further_topics/ugrid/operations.rst

```diff
@@ -502,50 +518,29 @@ earlier:
         ...         start_index=face_node.start_index,
         ...     )
         ...     return mesh
-
-        >>> print(face_cube)
-        face_data / (K)                     (-- : 2; height: 3)
-            Dimension coordinates:
-                height                          -          x
-            Mesh coordinates:
-                latitude                        x          -
-                longitude                       x          -
-            Attributes:
-                Conventions                 'CF-1.7'
-
-        # Convert our mesh+data to a PolyData object.
-        # Just plotting a single height level.
-        >>> face_polydata = cube_faces_to_polydata(face_cube[:, 0])
+        ...
+        >>> face_polydata = cube_faces_to_polydata(cube)
         >>> print(face_polydata)
-        PolyData (0x7ff4861ff4c0)
-          N Cells:	2
-          N Points:	5
-          X Bounds:	9.903e-01, 1.000e+00
-          Y Bounds:	0.000e+00, 1.392e-01
-          Z Bounds:	6.123e-17, 5.234e-02
-          N Arrays:	2
-
-        # Create the GeoVista plotter and add our mesh+data to it.
+        PolyData (0x7f179a6b2800)
+          N Cells:    96
+          N Points:   98
+          N Strips:   0
+          X Bounds:   -1.000e+00, 1.000e+00
+          Y Bounds:   -1.000e+00, 1.000e+00
+          Z Bounds:   -1.000e+00, 1.000e+00
+          N Arrays:   4
+        >>>
         >>> my_plotter = GeoPlotter()
-        >>> my_plotter.add_coastlines(color="black")
-        >>> my_plotter.add_base_layer(color="grey")
-        >>> my_plotter.add_mesh(face_polydata)
-
-        # Centre the camera on the data.
-        >>> camera_region = to_xyz(
-        ...     face_cube.coord("longitude").points,
-        ...     face_cube.coord("latitude").points,
-        ...     radius=3,
-        ... )
-        >>> camera_pos = camera_region.mean(axis=0)
-        >>> my_plotter.camera.position = camera_pos
-
+        >>> my_plotter.background_color = "#502020ff"
```

Comment:
If you use `import geovista.theme` there's no need to set the `background_color`